wpeverest/everest-forms
# Feature - WPBakery Builder Compatibility

### All Submissions:






### Changes proposed in this Pull Request:

Previously, there was not inbuilt **Everest Forms** element widget for WPBakery builder. This PR adds the feature to use **Everest Forms** directly from the **WPBakery** builder.

Closes # .

### How to test the changes in this Pull Request:

1. You need to install the **WPBakery Pro**  
2. Create a Page using **WPBakery Editor**
3. Click on the + Icon and select the **Everest Forms** Tab
4. You can view the **Registration Forms**
5. Select the desired form from the list and check whether it is working accordingly or not.

### Types of changes:




### Other information:




### Changelog entry

>  Feature -  WPBakery Builder Compatibility
